app/models/company.rb

```diff
@@ -1,2 +1,7 @@
 class Company < ActiveRecord::Base
+  validates :name,            presence: true
+  validates :description,     presence: true
+  validates :sub_description, presence: true
```

Comment:
ok, how about name, title, description instead of name, description sub_description?